src/test/java/com/thealgorithms/dynamicprogramming/TravelingSalesManPoblem.java

```diff
@@ -0,0 +1,61 @@
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class TSPTest {
```

Comment:
It's better to use `@ParameterizedTest`. Also, it is better to test both the path and minimum cost for all cases.